dagmc/dagnav.py

```diff
@@ -589,16 +629,28 @@ def merge(self, other_group):
 
     @classmethod
     def create(cls, model: DAGModel, name: Optional[str] = None, group_id: Optional[int] = None) -> Group:
-        """Create a new group instance with the given name"""
+        """Create a new group instance with the given name, 
+        or return an existing group if one exists."""
+
+        # return existing group if one exists with this name
+        if name is not None:
```

Comment:
I'm not sure what to do with Group IDs. Part of me thinks we should notify if the `group_id` paramter is set and it doesn't match the returned `Group.id` if the group already exists. Functionally, the Group IDs really don't matter to DAGMC, but it still makes them complete in the eyes of MOAB. Thoughts?

Reply:
I was on the fence about this, too.  I'm happy to add it - it's pretty easy. I'll see what others have to say....